src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const obj of actions) {
+    if (obj.type === 'addProperties') {
+      Object.assign(state, obj.extraData);
+    }
```

Comment:
Checklist 2. [CODE STYLE]: Use switch statement if you have limited amount of conditions.

Replace if with switch